app/views/coronavirus_landing_page/components/shared/_timeline.html.erb

```diff
@@ -1,23 +1,90 @@
 <div class="covid-timeline" data-module="track-links" data-track-category="pageElementInteraction" data-track-action="Timeline">
   <%= render "govuk_publishing_components/components/heading", {
-    text: timeline["heading"],
+    text: details.timeline["heading"],
     padding: true,
-    margin_bottom: 4
+    margin_bottom: 2
   } %>
 
-  <% if timeline["summary"] %>
-    <%= render_govspeak(timeline["summary"]) %>
-  <% end %>
+  <% if details.show_timeline_nations? %>
+    <%= form_tag coronavirus_landing_page_path,
+      method: :get,
+      class: "js-change-location" do |f| %>
+
+      <p class="govuk-body">
+        <%=t("coronavirus_landing_page.show.timeline.controls.status") %>
+        <span class="js-current-nation covid-timeline__nation"><%= @selected_nation.titleize %></span>.
```

Comment:
I wanted to give it an `aria-live=polite` so that screen readers would announce the text when the radio button is changed, but it turns out that some of them only announce what has changed, so for example Voiceover would announce a lot of things, including simply the word 'England' (or Scotland, etc.), instead of the full text (which would be more helpful) of 'Showing updates for England'. This would get lost inside the other things it was saying, including the word England as the thing that had been selected - it sounded like a weird repetition.

So I've put the aria-live attribute onto the actual content instead, and there's visually hidden headings so the first things users should hear is the full 'Updates for England' when they change the radio button.